test/integration.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-env mocha */
```

Comment:
I don't know if `standard` supports this, but at least with normal ESLint, if you want to enable the `mocha` environment for the entire `test/` directory instead of file-by-file, you can add `test/.eslintrc.json` with `{ "env": { "mocha": true } }`.